packages/react-components/src/components/styles/SendBox.styles.ts

```diff
@@ -31,14 +31,17 @@ export const sendBoxStyle = mergeStyles({
  * @private
  */
 export const sendButtonStyle = mergeStyles({
-  margin: '.6rem'
```

Comment:
(on this iteration, looks to have removed the marginLeft and marginRight)

Reply:
should be changed now.

Reply:
Now we've increased the width the icon takes up by ~0.2rem, can we reduce this margin, looks to have a large right margin between the icon and send box edge:
![image](https://user-images.githubusercontent.com/2684369/142037931-1bc6dfbf-b07f-4b14-979f-9ba2b38afd3a.png)

Reply:
So the correct change is to do something like marginLeft .6rem and marginRIght .4rem? @JamesBurnside